checks/net.go

```diff
@@ -192,6 +193,23 @@ func formatDirection(d ebpf.ConnectionDirection) model.ConnectionDirection {
 	}
 }
 
+func formatConntrack(ct *ebpf.Conntrack) *model.Conntrack {
+	if ct == nil {
```

Comment:
Just wanted to confirm that the idea around this is that if there is no address translation, that we don't send anything to the backend. 

Is that accurate?

Reply:
It's not, but it would be pretty easy to filter out the entries that don't have any translation.

Reply:
I think there's value in doing this to reduce message sizes. As a TODO for later :)

Reply:
It was actually pretty easy to add. All the calls to `isNAT` are guarding against saving the non-NATd entries.